docs/notebooks/tremor/tremor_analysis.ipynb

```diff
@@ -119,7 +119,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.8"
+   "version": "3.11.5"
```

Comment:
Python seems to be downgraded in your notebooks, you should probably update your Poetry.